src/collections/_documentation/workflow/releases/release-automation/travis-ci/index.md

```diff
@@ -0,0 +1,67 @@
+
+[{% asset releases/travis-ci/internal-integration-permissions.png alt="View of internal integration permissions." %}]({% asset releases/travis-ci/internal-integration-permissions.png @path %})
+
+Click "Save", then record your token, which you'll need in the next section.
+
+## Setting Environment Variables in Travis CI
+
+Next, you'll need a few environment variables to configure the Sentry CLI:
+
+- `SENTRY_AUTH_TOKEN` - Your internal integration token
+- `SENTRY_ORG` - Your Sentry organization slug
+- `SENTRY_PROJECT` - Your Sentry project slug
+
+To access you internal integration token securely, store it as an [environment variable on your repository](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings):
+
+1. In the Travis CI application, go to your repository's settings by clicking "Settings" from the "More options" menu.
+2. Find the "Environment Variables" section.
+3. Add a new variable with `SENTRY_AUTH_TOKEN` as the name and your internal integration token as the value.
+
+[{% asset releases/travis-ci/travis-env-vars.png alt="View of Travis CI environment variables." %}]({% asset releases/travis-ci/travis-env-vars.png @path %})
+
+The other variables do not contain sensitive information, so you'll use the `env` key to define them as environment variables in your .travis.yml file.
+
+## Create Release and Notify Sentry of Deployment
+
+To automate your Sentry release management process you'll need to add the `Create release and notify Sentry of deploy` job below after your deploy step:
+
+```yaml
+jobs:
+    include:
+# ...
+      - name: Create release and notify Sentry of deploy
+        env: SENTRY_ORG=sample-organization-slug SENTRY_PROJECT=sample-project-slug SENTRY_ENVIRONMENT=production
+        script: |
+          curl -sL https://sentry.io/get-cli/ | bash
+          export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+          sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
+          sentry-cli releases set-commits $SENTRY_RELEASE --auto
+          sentry-cli releases files $SENTRY_RELEASE upload-sourcemaps path-to-sourcemaps-if-applicable
+          sentry-cli releases finalize $SENTRY_RELEASE
+          sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_ENVIRONMENT
+```
+
+For more details about the release management concepts in the snippet above, see the full documenation on [release management]({%- link _documentation/cli/releases.md -%}).
+
+**Notes**:
+
+- If you’re not deploying a JavaScript project or have sent source maps to Sentry using another method, omit the `upload-sourcemaps` line.
+- If you can’t install a repository integration, send commit metadata via the [create release endpoint]({%- link _documentation/workflow/releases/index.md -%}#alternatively-without-a-repository-integration) or omit the `set-commits` line.
```

Comment:
I feel like we should tell folks that if they omit set-commits, they won't get suspect commits. I know we allude to that up top but I feel like also putting it here makes it clear on what they'll be missing out. Thoughts?